nbs/methods.ipynb

```diff
@@ -44,6 +44,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.                      sparsity: Optional[bool] = True):
</pre><p><code>sparsity</code> could be an attribute of the class instead of an extra argument. If <code>self.sparsity=True</code> , then <code>P</code> and <code>S</code> could be sent to <code>_reconcile</code> as sparse matrices.      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
My thought on skipping the `self.sparsity` attribute is to avoid extra complexity to the methods.  Some reconciliation methods might report gains consistently while other don't. 

We might want to make the selection a default.